lib/build.js

```diff
@@ -10,6 +10,7 @@ const apiUri = Symbol('apiUri');
 const tokenGen = Symbol('tokenGen');
 const uiUri = Symbol('uiUri');
 const ABORT_CODE = 130; // 128 + SIGINT 2 (^C)
+const TEMPORAL_JWT_TIMEOUT = 120 * 60; // 120 minutes in seconds
```

Comment:
I thought this was going to be 12hrs so that it's inline with `MAX_EXPIRES_IN`

Reply:
Yes, you are right. I fixed to 12hours.